src/map/script.c

```diff
@@ -12284,36 +12284,69 @@ static BUILDIN(monster)
  *------------------------------------------*/
 static BUILDIN(getmobdrops)
 {
-	int class_ = script_getnum(st,2);
-	int i, j = 0;
-	struct mob_db *monster;
+	struct map_session_data *sd = NULL;
+	int mob_id = script_getnum(st, 2);
+	struct mob_db *monster = NULL;
+	struct script_data *data1 = script_getdata(st, 3);
+	struct script_data *data2 = NULL;
+	const char *varname1 = reference_getname(data1);
+	const char *varname2;
+	int varid1 = reference_getid(data1);
+	int varid2;
```

Comment:
Might be safer to initialize these even if they are not used in some cases

```suggestion
	const char *varname2 = NULL;
	int varid1 = reference_getid(data1);
	int varid2 = 0;
```